pango/src/auto/font.rs

```diff
@@ -27,6 +33,28 @@ glib::wrapper! {
 
 impl Font {
     pub const NONE: Option<&'static Font> = None;
+
+    #[cfg(any(feature = "v1_50", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v1_50")))]
+    #[doc(alias = "pango_font_deserialize")]
+    pub fn deserialize(
+        context: &Context,
+        bytes: &glib::Bytes,
+    ) -> Result<Option<Font>, glib::Error> {
```

Comment:
This is not actually `nullable`, is it?

Reply:
it seems to be nullable https://gitlab.gnome.org/GNOME/pango/-/blob/main/pango/serializer.c#L1543